akka-docs-dev/rst/java/stream-error.rst

```diff
@@ -0,0 +1,69 @@
+.. _stream-error-java:
+
+##############
+Error Handling
+##############
+
+Strategies for how to handle exceptions from processing stream elements can be defined when
+materializing the stream. The error handling strategies are inspired by actor supervision
+strategies, but the semantics has been adapted to the domain of stream processing.
+
+Supervision Strategies
+======================
+
+There are three ways to handle exceptions from application code:
+
+* ``Stop`` - The stream is completed with failure.
+* ``Resume`` - The element is dropped and the stream continues.
```

Comment:
place a link to the graphs and deadlocks section and mention that dropping can result in deadlocks in certain scenarios, for example broadcast -> flow -> zip